ql/src/test/queries/clientpositive/annotate_stats_lateral_view_join.q

```diff
@@ -0,0 +1,38 @@
+set hive.fetch.task.conversion=none;
```

Comment:
To make EXPLAIN show Statistics. I'm thinking to create another ticket and add this line to other `annotate_stats_*.q`.

e.g. https://github.com/apache/hive/blob/master/ql/src/test/results/clientpositive/llap/annotate_stats_select.q.out